docs/content/usage/agit-support.en-us.md

```diff
@@ -26,24 +27,25 @@ This can be done by pushing to the branch followed by a specific refspec (a loca
 The following example illustrates this:
 
 ```shell
-git push origin HEAD:refs/for/master
+git push origin HEAD:refs/for/main
 ```
 
 The command has the following structure:
 
-- `HEAD`: The target branch
-- `refs/<for|draft|for-review>/<branch>`: The target PR type
+- `origin`: The target repository (not a fork!)
+- `HEAD`: The local branch containing the changes you are proposing
+- `refs/<for|draft|for-review>/<branch>`: The target PR type and configuration
   - `for`: Create a normal PR with `<branch>` as the target branch
-  - `draft`/ `for-review`: Currently ignored silently
-- `<branch>/<session>`: The target branch to open the PR
+  - `draft`/`for-review`: Currently ignored silently
+  - `<branch>/`: The branch you want your changes to be merged into
 - `-o <topic|title|description>`: Options for the PR
-  - `title`: The PR title
-  - `topic`: The branch name the PR should be opened for
-  - `description`: The PR description
+  - `topic`: The branch name the PR should be opened for, this is REQUIRED to trigger a pull request.
```

Comment:
This is not branch name in fact. The previous document is also inaccurate. It will be displayed in PR as a source, the place will be the same as Github Flow's branch name.

Reply:
See how you like  97c6593e2